src/EditProvider.ts

```diff
@@ -112,8 +113,10 @@ export class EditProvider
     searchStartPath: string | undefined = vscode.workspace.rootPath
   ): Promise<LanguageOptionValues> {
     try {
-      const vscodeSettings = vscode.workspace.getConfiguration("unibeautify");
-      const defaultConfigFile = vscodeSettings.defaultConfig;
+      const vscodeSettings: UnibeautifyVSCodeSettings = <any>vscode.workspace.getConfiguration(
+        "unibeautify"
+      );
+      const defaultConfigFile = (vscodeSettings as any).defaultConfig;
```

Comment:
Should not need the casting to `any`:

```diff
-      const defaultConfigFile = (vscodeSettings as any).defaultConfig;
+      const defaultConfigFile = vscodeSettings.defaultConfig;
```

Add `defaultConfig` to interface at https://github.com/muuvmuuv/vscode/blob/f74329666cbc0b647afdfccf2f100a52e92a4576/src/index.ts#L67

Reply:
Ah okay :)